lib/pgp.ml

```diff
@@ -396,6 +399,7 @@ module Packet = struct
     let decode packet =
       let version = Cstruct.get_uint8 packet 0 in
       let creation_time = Cstruct.BE.get_uint32 packet 1 in
+
```

Comment:
Extra empty line.

Reply:
Oh, I thought that ocamlformat would fix automatically things like that!